layouts/base.pug

```diff
@@ -135,6 +135,15 @@ html(lang='en')
                     gcse.src = (document.location.protocol == 'https:' ? 'https:' : 'http:') + '//cse.google.com/cse.js?cx=003287487325783809005:' + cseId;
                     var s = document.getElementsByTagName('script')[0];
                     s.parentNode.insertBefore(gcse, s);
+                    window.onload = function() {
+                        document.getElementById('gsc-i-id1').placeholder = "Type 'F' to Search";
+                    }
+                    onkeyup = function(e){
+                        var searchInput = document.getElementById('gsc-i-id1');
+                        if(e.keyCode==70 && e.target.tagName !== 'INPUT'){
```

Comment:
Additionally, I think we also want to prevent cases where `tagName === 'TEXTAREA'` since we have that ProdPad widget now.

Nitpick: spaces before and after the `===` and before the opening code block `{` character.

Reply:
good catch 🙏